packages/scandipwa/src/component/FieldDate/FieldDate.container.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+import PropTypes from 'prop-types';
+import { PureComponent } from 'react';
+import { connect } from 'react-redux';
+
+import DatePicker from 'Component/DatePicker';
+import DateSelect from 'Component/DateSelect';
+
+import { FIELD_DATE_TYPE } from './FieldDate.config';
+
+/** @namespace Component/FieldDate/Container/mapStateToProps */
+export const mapStateToProps = (state) => ({
+    useCalendar: !!+state.ConfigReducer.use_calendar
```

Comment:
btw, this one you can actually just set boolean in schemas.graphql so it will convert 0 and 1 into false and true.

Reply:
Done: https://github.com/scandipwa/store-graphql/pull/31
Thanks, didn't know it can be done in such an easy way on BE.

I didn't change FE code for now, will change after BE PR is merged in order that FE works and can be tested now.
(`!!+"1" = true` and `!!+true = true` => before and after BE PR merge FE part will work as expected).

Reply:
Yea, it's anyway great code as always